radar_patient_check/main.py

```diff
@@ -1,49 +1,94 @@
-import os
+from datetime import date
+from pydantic import BaseModel, Field
 
 from fastapi import Depends, FastAPI, HTTPException
 from fastapi.security import OAuth2PasswordBearer
-from sqlmodel import Session
-from ukrdc_sqla.ukrdc import PatientNumber, ProgramMembership
+from ukrdc_sqla.ukrdc import Patient, PatientNumber, ProgramMembership
 
-from radar_patient_check.database import ukrdc_engine
+from .database import get_session
+from .config import settings
 
 oauth2_scheme = OAuth2PasswordBearer(tokenUrl="token")
 
 app = FastAPI()
 
 
+class RadarCheckRequest(BaseModel):
+    nhs_number: str = Field(
+        ..., description="The patient's NHS number", alias="nhsNumber"
+    )
+    date_of_birth: date = Field(
+        ..., description="The patient's date of birth", alias="dateOfBirth"
+    )
+
+
+class RadarCheckResponse(BaseModel):
```

Comment:
Use a proper Pydantic response model to validate output, and keep code a bit cleaner.